src/app/auth/forgotPassword/index.tsx

```diff
@@ -25,7 +25,11 @@ function ForgotPasswordScreen() {
     const { error } = await resetPassword(emailToReset);
     if (error)
       Alert.alert('Could not send a reset password email. Please try again.');
-    else router.replace('auth/signup');
+    else
+      router.push({
+        pathname: '/auth/verify',
+        params: { finalRedirect: 'resetPassword', emailAfterRedirect: email },
```

Comment:
i would rename `emailAfterRedirect` to something more descriptive, maybe like `userEmail`? up to you!

Reply:
also, make sure to use `emailToReset` as the value of the param because `email` might be the username of the user, while `emailToReset` will always be an email.